src/db/init_database.js

```diff
@@ -39,12 +48,26 @@ export async function populateTools() {
     usageRequirements: 'You must have created a Heroku account before using this tool.'
   });
   await sequelize.Tool.create({
-    name: 'Express.js',
+    name: 'ExpressJS',
     toolType: sequelize.ToolType.WEB_FRAMEWORK,
     websiteUrl: 'http://expressjs.com/',
     documentationUrl: 'http://expressjs.com/en/4x/api.html',
     logoLargeUrl: 'https://camo.githubusercontent.com/fc61dcbdb7a6e49d3adecc12194b24ab20dfa25b/68747470733a2f2f692e636c6f756475702e636f6d2f7a6659366c4c376546612d3330303078333030302e706e67',
   });
+  await sequelize.Tool.create({
+    name: 'Mocha',
+    toolType: sequelize.ToolType.TEST,
+    websiteUrl: 'https://mochajs.org/',
+    documentationUrl: 'https://mochajs.org/#getting-started',
+    logoLargeUrl: 'https://cdn.worldvectorlogo.com/logos/mocha-1.svg'
+  });
+  await sequelize.Tool.create({
+    name: 'Sequelize',
+    toolType: sequelize.ToolType.DATABASE,
```

Comment:
This should maybe be `DATABASE_ORM`, since Sequelize is just an ORM. Within Sequelize, you have different dialects for your database, such as mysql, sqlite, postgres, etc. Not saying you have to specify those right now, I'm just saying it will make things clearer.

Reply:
I did it this way to stay consistent with tyr. I think that we could make a change in tyr and do it here, however until then, this should stay.

Reply:
Ok whatever. Not make or break at this point.